sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyTitle>Azure ServiceBus SDK</AssemblyTitle>
-    <Description>This is the next generation Azure Service Bus .NET Standard client library that focuses on queues &amp; topics. For more information about Service Bus, see https://azure.microsoft.com/en-us/services/service-bus/</Description>
-    <Version>5.2.0-beta.1</Version>
+    <Description>Please note, a newer package Azure.Messaging.ServiceBus is available as of November 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/net/migrate/sb for more details.</Description>
```

Comment:
```suggestion
    <Description>Please be aware that a newer package, Azure.Messaging.ServiceBus, was released in November 2020. While this package will continue to receive critical bug fixes, we strongly encourage upgrading for new development. Please see the migration guide at https://aka.ms/azsdk/net/migrate/eh for more details.</Description>
```

Reply:
In my opinion, the `for new development` is a significant distinction.  While we'd like to see migration in existing applications, I don't believe there is forcing function for it nor that there is enough of a value proposition to justify the efforts for many applications with an existing investment in the legacy packages.

Reply:
I believe @elraikhm went through an approval process for the current message. @elraikhm, would you be ok with @jsquire's recommendation above?

Reply:
@jsquire, I understand your reasoning, but we would like to recommend upgrading to the new package when possible for existing apps as well.  The app can live for years and customers will need to upgrade eventually to stay current and to receive the latest features and fixes.